config/webpack/loaders.js

```diff
@@ -5,6 +5,14 @@
 
 'use strict';
 
+var es6Loader;
+
+if (process.env.JASMINE) {
+    es6Loader = 'babel?plugins[]=rewire&plugins[]=object-assign';
```

Comment:
this looks off. How did you figure out the syntax of adding the `plugins[]` parameter twice? I would think it should look something like `babel?plugins=rewire,object-assign` or something like that.